### ISSUE TITLE
The "Compile with PL/Scope" action should preserve prior PL/SQL compilation settings

### ISSUE THREAD

**Issue by rvo-cs** [PhilippSalvisberg/plscope-utils#58]
As of v1.0.0, the "Compile with PL/Scope" action changes the `plscope_settings` parameter in the session, then recompiles objects in the schema (unless the user has opted out of that), enforcing whatever PL/SQL compilation parameters are effective in the session when doing so.

The expectation is that it would preserve prior PL/SQL compilation settings for all affected objects, with the single exception of updating the `plscope_settings` parameter.

This is important because some modules could have specific requirements as regards compilation settings (especially, but not limited to, the value of the `plsql_ccflags` parameter), and these settings should not be lost when running the action.

**Comment by PhilippSalvisberg** [comment]
I agree. The existing settings (except identifiers/statements) should be preserved.
